docs/v2.4/reference/rbac.md

```diff
@@ -41,7 +41,79 @@ After creating `config.yaml` in your current directory, export the file's locati
 
 ## Administrating Telepresence
 
-[Telepresence administration](/products/telepresence/) requires permissions for creating `Namespaces`, `ServiceAccounts`, `ClusterRoles`, `ClusterRoleBindings`, `Secrets`, `Services`, `MutatingWebhookConfiguration`, and for creating the `traffic-manager` [deployment](../architecture/#traffic-manager) which is typically done by a full cluster administrator.
+Telepresence administration requires permissions for creating `Namespaces`, `ServiceAccounts`, `ClusterRoles`, `ClusterRoleBindings`, `Secrets`, `Services`, `MutatingWebhookConfiguration`, and for creating the `traffic-manager` [deployment](../architecture/#traffic-manager) which is typically done by a full cluster administrator. The following permissions are needed for the instillation and use of Telepresence.
+
+```yaml
```

Comment:
Have you tested that using a user with this RBAC successfully enables you to install the `traffic-manager` into the cluster? Just because whenever we put yaml on the website, you can bet someone is going to copy+paste and try it. So we should be certain it works as is.

Reply:
Yes, I have run this through a smoke test and a webhook test